src/pb_stub.cc

```diff
@@ -402,7 +470,40 @@ Stub::Initialize(bi::managed_external_buffer::handle_t map_handle)
           .attr("TritonPythonModel");
   deserialize_bytes_ = python_backend_utils.attr("deserialize_bytes_tensor");
   serialize_bytes_ = python_backend_utils.attr("serialize_byte_tensor");
-  model_instance_ = TritonPythonModel();
+
+  return TritonPythonModel();
+}
+
+void
+Stub::AutoCompleteModelConfig(
+    bi::managed_external_buffer::handle_t string_handle,
+    std::string* auto_complete_config)
+{
+  model_ = StubSetup();
```

Comment:
Why are we constructing the instance here? Since the auto_complete_config method is a static method we can call this method without constructing an object.

Reply:
Since the `auto_complete_config` is a static method we don't need to instantiate the object. I think we can just use the `py::hasattr(TritonPythonModel, "auto_complete_config")`.